### PR TITLE
[DUOS=2244][risk=low] UX Followons for info page

### PR DESCRIPTION
**Pull request by nmalfroy**
- Update images for size and color
- Fix some text
- Make pages responsive
- Add alt text to images

Check out the behavior for the various sections.  At some point, upgrading to Material UI 5 (e.g. MUI) makes this easier

Top of page
top-resize

Middle sections
middle-resize

Bottom bullet points
bottom-resize

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/DMSPolicyInfo.css [hunk 1]
```diff
@@ -49,19 +59,43 @@
 .subtitle-section .section-text {
   padding-top: 10px;
 }
+/*
+  This means that when the window is less than 80px wide (e.g. grid breakpoint = l) then the laptop image will be hidden
```
Comment (fboulnois) [inline src/pages/DMSPolicyInfo.css]:
```suggestion
  This means that when the window is less than 800px wide (e.g. grid breakpoint = l) then the laptop image will be hidden
```

File: src/pages/DMSPolicyInfo.css [hunk 2]
```diff
@@ -49,19 +59,43 @@
...
 .store-data-section {
   background-color: #ececec; /* d8d8d8d with 0.5 opacity over white */
   padding: 50px 0;
 }
 .store-data-section img {
   width: 100%;
-  padding-left: 40px;
 }
-
+/* remove padding in mobile mode */
```
Comment (fboulnois) [inline src/pages/DMSPolicyInfo.css]:
```suggestion
/* add padding in desktop view */
```

I would either put `/* no padding in mobile view */` on the `.store-data-section` without media queries, or I would recommend using my suggestion instead. As is it seems a bit confusing to have the comment above the media query. Ditto for the other cases.

**Review by fboulnois — COMMENTED** [review]
Some minor suggestions. Also do we use `svg`s anywhere in the UI? If so the duos chart seems like a prime candidate, especially since it means we can losslessly resize it on different viewports.

**Review by rushtong — APPROVED** [review]
Looked at it in mobile mode, looks great 👍🏽

File: src/pages/DMSPolicyInfo.css (hunk 1 above)
Comment (nmalfroy) [reply src/pages/DMSPolicyInfo.css]: gah, nice catch

File: src/pages/DMSPolicyInfo.css (hunk 2 above)
Comment (nmalfroy) [reply src/pages/DMSPolicyInfo.css]: ah yeah, that's a fair point.  Will update

**Comment by nmalfroy** [comment]
Yeah, there are SVGs but I just used what I got from Lou for this one

**Review by fboulnois — APPROVED** [review]
lgtm 😎

**Pull request merged**